tests/test_transforms/test_math/test_limit_transform.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from etna.datasets import TSDataset
+from etna.transforms.math import LimitTransform
+from etna.models import ProphetModel
+from etna.pipeline import Pipeline
+
+
+
+@pytest.fixture()
+def example_ts_(example_df_with_nontarget_column) -> TSDataset:
+    ts = TSDataset(example_df_with_nontarget_column, freq="D")
+    return ts
+
+
+@pytest.mark.parametrize("lower_bound,upper_bound", [(-1e10, 1e10), (-15, 100)])
+def test_fit_transform_target(example_ts_: TSDataset, lower_bound: float, upper_bound: float):
```

Comment:
Typings here are not obligatory as mypy does't check it. You can add them if you want